telemetrylib/UDP.cpp

```diff
@@ -0,0 +1,45 @@
+//
+// Created by Mingcan Li on 1/30/24.
+//
+
+#include "DTI.h"
+
+class UDP : public DTI {
+public:
+    UDP(const QHostAddress& serverAddress, int serverPort) {
+        //_udpSocket.bind(localAddress, localPort);
+        this->serverAddresses = serverAddress;
+        this->udpPort = serverPort;
+        _udpSocket = new QUdpSocket();
+        emit connectionStatusChanged();
+    }
+
+    ~UDP() {
+        // Cleanup and closing of the UDP socket can be added here if needed.
+    }
+
+    void sendData(QByteArray bytes, long long timestamp) override {
+        qDebug() << "sending via UDP";
+        bytes.prepend("<bsr>");
+        bytes.append("</bsr>");
+        qDebug() << _udpSocket->writeDatagram(bytes, serverAddresses, udpPort);
```

Comment:
Is this debug print meant to stay or should it be taken out before this gets merged?